src/redux/books/books.js

```diff
@@ -1,16 +1,49 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/UULDvvE4EWqYt0Y5rD8Z/books',
+// {
+//   item_id: 2,
+//   title: "Harry Potter",
+//   author: "JK Rowling",
+//   category: "Fiction"
+// }
+// )
+//   .then((response) => console.log(response.status));
+
+// axios.delete('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/UULDvvE4EWqYt0Y5rD8Z/books/')
+// .then(response => console.log(response.data))
```

Comment:
- [ OPTIONAL ] I think it would be great if we remove the commented code to make your code clean and dry so please remove it 🚀🔥😊💪🏻

Reply:
Thanks @Reem-lab, i will do that. you have really been a nice code reviewer